src/gwt/src/org/rstudio/core/client/patch/DiffMatchPatch.java

```diff
@@ -1,58 +0,0 @@
-/*
- * DiffMatchPatch.java
- *
- * Copyright (C) 2009-12 by RStudio, Inc.
- *
- * Unless you have received this program directly from RStudio pursuant
- * to the terms of a commercial license agreement with RStudio, then
- * this program is licensed to you under the terms of version 3 of the
- * GNU Affero General Public License. This program is distributed WITHOUT
- * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
- * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
- * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
- *
- */
-package org.rstudio.core.client.patch;
-
-import com.google.gwt.core.client.GWT;
-import com.google.gwt.dom.client.Document;
-import com.google.gwt.dom.client.HeadElement;
-import com.google.gwt.dom.client.ScriptElement;
-import com.google.gwt.resources.client.ClientBundle;
-import com.google.gwt.resources.client.TextResource;
-
-public class DiffMatchPatch
-{
-   interface Resources extends ClientBundle
-   {
-      @Source("diff_match_patch.js")
```

Comment:
I wonder if this means we no longer require `diff_match_patch.js`?

Reply:
Ah maybe, didn't notice that part. I'll take a peek when I get a chance.